snippets/cart-drawer.liquid

```diff
@@ -0,0 +1,11 @@
+{% comment %}
+    Renders cart drawer
+
+    Accepts:
+    - color_scheme: {String} sets the color scheme of the notification (optional)
```

Comment:
Looks like this was copied from `cart-notification.liquid` and we don't accept this parameter here.

Reply:
Oh this is because it was part of the design but will need to be added in a different setting for more than just the cart drawer. Thanks for flagging tho, ill remove it for now.

Here are the designs for context: Colour schemes. [Figma.](https://www.figma.com/file/JcYdswrk2JhqBTGBVWPM7J/Dawn-2.0?node-id=13884%3A111446) |  Other drawers colour schemes. [Figma]